include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -110,7 +111,7 @@ A_{3, 2}  A_{3, 3}  A_{3, 0}  A_{3, 1} ...   [phase 1] /
             // 4. TODO: what about f64?
             //
             // maxPhase is set to SIMDWidth / perPhase
-            int vecSize = (eltTy.isF16() ? 64 : 32 ) / typeBitWidth;
+            int vecSize = ((typeWidthInBit == 16) ? 64 : 32 ) / typeBitWidth;
```

Comment:
Just realized that this was originally checking explicitly for fp16 to set the vecsize to 4. Now with this new typeWidthInBit variable that will also catch bfloat16, which I though we wanted at vecsize=2.  Please check me on this.

Reply:
We designed vecSize as the number of elements per thread per mfma instruction. From this perspective, bf16 needs vecsize=2 on MI100 but 4 on MI200.
However, setting vecsize=4 on MI100 should not lead to wrong results. And I doubt if it leads to worse performance on MI100, since vecSize only controls the vector length when we write to LDS. But mfma operand (here this is only for k-major matrices) can take the correct number of elements for each thread. 
In addition, MI100 is not the focus right now. And we'll revisit this swizzling stuff in the future after we get other "bigger" optimization done. So I think this code is good to go. @binarman what do you think?

Reply:
@zhanglx13 
You are right, `vec size` greater than `elements per thread` should not bring any performance penalty on mi100 (and any other GPU, actually), if this size is sufficient to eliminate bank conflicts.

Few words about correctness. `Shared` -> `mfma dot operand` conversion is designed in a way it produces correct code for **any** valid vecSize. The only thing affected by vec size is performance: with small vec size we can not do large vector loads.